addons/sale/models/sale_order.py

```diff
@@ -13,11 +13,17 @@
 
 class SaleOrder(models.Model):
     _name = "sale.order"
-    _inherit = ['portal.mixin', 'mail.thread', 'mail.activity.mixin', 'utm.mixin']
+    _inherit = ['portal.mixin', 'mail.thread', 'mail.activity.mixin', 'utm.mixin', 'sequence.mixin']
```

Comment:
I tried quickly to find a reason/explanation/message to know why this mixin is needed.
It's not even used in subscription, why would we need to use it here?
(if we can avoid more `account` mess in sale, I'd prefer to avoid it :D)

Reply:
It is coming from the spec. The idea is to remove the ir.sequence used to get the so name.
it allows to define a "prefix" in the sale_order_template and have easily different so name depending on the used template. It's an FP request.

Reply:
It might be a FP request, but it's literally destroying any setup sequence-based.
Honestly, we should find a way to support both cases together, even if the default behavior does not rely on ir.sequence anymore.